concent_api/command_line_tool/message_extractor.py

```diff
@@ -0,0 +1,122 @@
+from base64 import b64decode
+import importlib
+import re
+import random
+import string
+from typing import Dict, Text, Any, List, Union
+from golem_messages.datastructures import FrozenDict
+from golem_messages.message import Message
+
+JsonType = Dict[Text, Any]
+Task = FrozenDict
```

Comment:
This is not needed.

Reply:
Added for convenience

Reply:
I understand the `JsonType = Dict[Text, Any]` but where is convenience in `Task = FrozenDict` ? It is actually one object assigned to another, adding only blur. Why not using directly the `FrozenDict` ?

Reply:
@rwrzesien, I actually like having this alias (`Task`) for `FrozenDict`. On concept level in golem/concent we have messages and tasks/subtaks. `FrozenDict` is actual implementation (low level detail) of task. Knowing that some function returns `Task` or `Message` is more helpful, than stating that it returns `FrozenDict` or `Message`.
I would be for leaving it in the code.

Reply:
Ok, that makes sense.